static/js-src/cs-client.js

```diff
@@ -204,6 +204,10 @@ class ChromeStatusClient {
   getChannels() {
     return this.doGet('/channels');
   }
+
+  getSpecifiedChannels(start, end) {
+    return this.doGet(`/channels?start=${start}&&end=${end}`);
```

Comment:
"&&" -> "&".

Reply:
Oops... Thank for pointing this out. 
Done. See [this](https://github.com/GoogleChrome/chromium-dashboard/pull/1458/commits/3d061ae21f43b4a5b1581d729418a29c769068eb) commit.